util/custom-response/src/main/java/com/brother/customresponse/repository/StudentRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.brother.customresponse.repository;
+
+import com.brother.customresponse.entity.Student;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+import java.util.Optional;
+
```

Comment:
We can use @Repository annotation here.

Reply:
> We can use @repository annotation here.

It is indeed not necessary to put the @Repository annotation on interfaces that extend JpaRepository; Spring recognizes the repositories by the fact that they extend one of the predefined Repository interfaces.
Ref info : [Medium](https://medium.com/java-epic/repository-not-necessary-when-extending-one-of-the-predefined-repository-interfaces-373b1773cced)